rubicon_ml/intake_rubicon/publish.py

```diff
@@ -4,6 +4,8 @@
 
 def publish(
     experiments,
+    # visualization object passed, defaulted to None 
+    visualization_object=None,
```

Comment:
We have been working to expand type hinting throughout the library, would request that we add mypy type hints to any parameters we are adding.

Reply:
So would the code look something like this? " visualization_object: Optional[ExperimentsTable] = None" and I import the experiments table from the viz library into publish.py?

Reply:
yup! if you don't need the `ExperimentsTable` anywhere other than the type hint, you can guard the import behind a `TYPE_CHECKING` block
> https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING

Reply:
that way, it'll only actually get imported when you're type checking. not at runtime where its not actually needed